examples/raytracing/example_raytracing.cpp

```diff
@@ -0,0 +1,224 @@
+/****************************************************************************
+ * Copyright (c) 2012-2021 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#include <ArborX.hpp>
+
+#include <Kokkos_Core.hpp>
+
+#include <iostream>
+#include <random>
+#include <vector>
+
+using ExecutionSpace = Kokkos::DefaultExecutionSpace;
+using MemorySpace = ExecutionSpace::memory_space;
+
+namespace ArborX
+{
+using Primitives = Kokkos::View<Box *, MemorySpace>;
+using Predicates = Kokkos::View<Ray *, MemorySpace>;
```

Comment:
This is not OK to put random things in ArborX:: namespace.  Specializing access traits is the only legal thing to do.